pkg/extract/extract.go

```diff
@@ -117,9 +119,18 @@ func untarFile(tarReader *tar.Reader, header *tar.Header, path string) error {
 	}
 	defer file.Close()
 
+	var reader io.Reader
+	reader = tarReader
+	if isatty.IsTerminal(os.Stdout.Fd()) && header.FileInfo().Size() >= 1_000_000_000 {
+		bar := pb.Simple.Start64(header.FileInfo().Size())
+		bar.Set("prefix", fmt.Sprintf("%s: ", filepath.Base(header.Name)))
```

Comment:
my main grief is that the bar is far too wide on fullscreen term, but that's a minor complaint, much better with one :)

Reply:
I set the max width parameter to 80. It will not fill the terminal anymore.